src/test/resources/application-test.yml

```diff
@@ -15,4 +15,10 @@ api:
     url: http://will-be-replaced-by-wireMockServer-bean
 
 dynamodb:
-  print-details-table-name: print-details
\ No newline at end of file
+  print-details-table-name: print-details
+  scheduler-lock-table-name: scheduler-locks
+
+jobs:
```

Comment:
Possibly for Integration testing, it may be better to call the method that the scheduler calls directly and to disable scheduled tasks in the test context (it might make testing easier, more predictable and faster than relying on a CRON to fire).

[An approach to disable scheduled tasks](https://stackoverflow.com/questions/61977922/is-there-a-global-flag-in-spring-boot-to-disable-all-the-scheduled-jobs):
```
@Configuration
@EnableScheduling
@ConditionalOnProperty(name="jobs.enabled", havingValue = "true")
public class ConditionalEnableScheduling {
}
```

Reply:
Yes, I totally agree with this. Said a similar thing to Nathan above